resources/default_pipeline_environment.yml

```diff
@@ -16,6 +16,9 @@ general:
       from: 'origin/master'
       to: 'HEAD'
       format: '%b'
+    rfc:
+      dockerImage: 'rfc'
```

Comment:
In fact providing a default is not that cool, since we will never deliver an image for that ...